src/components/checkboxes/index.md.njk

```diff
@@ -9,7 +9,7 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-Let users select one or more options by using the checkboxes component.
+Let people select one or more options by using the checkboxes component.
```

Comment:
The lowercase change looks great but I think you mistakenly changed 'users' to 'people' alongside this change.

```suggestion
Let users select one or more options by using the checkboxes component.
```